features/step_definitions/login_and_dashboard_steps.rb

```diff
@@ -1,40 +1,32 @@
-Then(/^I should see a dashboard page with my performance reports$/) do
+Then(/^I see a dashboard page with my performance reports$/) do
   current_time = Time.now
 
   Timecop.freeze(current_time) do
-    page = UI::Pages::Dashboard.new
-    page.displayed?
+    dashboard.displayed?
```

Comment:
I think here we should maybe use `expect(dashboard).to be_displayed`, so it's clear that it can also fail.